RecoLocalFastTime/FTLCommonAlgos/plugins/MTDRecHitAlgo.cc

```diff
@@ -39,6 +48,11 @@ FTLRecHit MTDRecHitAlgo::makeRecHit(const FTLUncalibratedRecHit& uRecHit, uint32
   float energy = 0.;
   float time = 0.;
 
+  // MTD topology
+  unsigned int index_topology = 0;  //1Disks geometry
```

Comment:
it looks like this index can be computed in `getEventSetup`, no need to re-derive it for every hit